tests/db/test_models.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from phoenix.db import models
+from sqlalchemy import select
+
+
+@pytest.mark.parametrize("session", ["sqlite", "postgres"], indirect=["session"])
+async def test_projects(session, project):
```

Comment:
```suggestion
async def test_projects(session):
```

Unnecessary fixture?

Reply:
we query for the project in the test, adding the fixture seeds the database with data